test/test-color-scheme.rb

```diff
@@ -78,5 +80,35 @@ def test_default
       assert_equal(expected_schema_keys.sort,
                    Test::Unit::ColorScheme.default.to_hash.keys.sort)
     end
+
+    def test_available_colors
+      terms = {
+        "rxvt"                 => 8,
+        "xterm-color"          => 8,
+        "alacritty"            => 256,
+        "iTerm.app"            => 256,
+        "screen-256color"      => 256,
+        "screenxterm-256color" => 256,
+        "tmux-256color"        => 256,
+        "vte-256color"         => 256,
+        "vscode-direct"        => 2**24,
+        "vte-direct"           => 2**24,
+        "xterm-direct"         => 2**24,
+      }
+      terms.each do |term, expected_available|
```

Comment:
We should not iterate test patterns.
We should use data driven feature for this: https://github.com/test-unit/test-unit/blob/master/test/test-data.rb#L40-L45

Reply:
I was just copy-paste coding and didn't know about that feature! I'll gladly convert to using it. :)